src/components/loginComponents/LoginForm.tsx

```
@@ -9,20 +10,50 @@ import { useFormState } from 'react-dom';
+      /^[가-힣a-zA-Z0-9]+$/,
+      '닉네임은 한글, 영문, 숫자만 사용할 수 있습니다.'
+    )
+});
+
+const initialState = { success: false, error: null };
+
+const LoginForm = ({ mode }: LoginFormProps) => {
   const router = useRouter();
   const { setUser } = useUserStore();
-  const [state, formAction] = useFormState(login, initialState);
+  const { data: userData } = useUserInfo();
```

ESLint

[error] 'userData' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)